src/app/demo-sect/demo-sect.component.html

```diff
@@ -194,6 +194,48 @@
   </div>
 </div>
 
+<div class="section free-tiles hero-tile bg-blue-dark">
+  <div class="row no-margin">
+    <div class="container">
+      <h2 class="text-center no-margin">Send any coins privately</h2>
```

Comment:
Don't know if we want this to have a margin or not. I removed it because it made the spacing look a bit weird imo, bu that might not be the case for you.